packages/web-components/scripts/type-check.js

```diff
@@ -1,12 +1,13 @@
 // @ts-check
 
 import fs from 'node:fs';
-import { fileURLToPath } from 'node:url';
 import path from 'node:path';
 import { promisify } from 'node:util';
 import { exec } from 'node:child_process';
 import { exit } from 'node:process';
 
+import { getDirname } from './utils.js';
```

Comment:
extracted this util to shared module as this pattern is used already in more than 1 place

Reply:
`__dirname` and `__filename` are available in Node >= 20.11.0 as `import.meta.dirname` and `import.meta.filename`

Reply:
nice TIL, i'll update , ty!